tests/backends/tests.py

```diff
@@ -745,6 +745,19 @@ def test_cursor_contextmanager_closing(self):
             self.assertIsInstance(cursor, CursorWrapper)
         self.assertTrue(cursor.closed)
 
+    @skipUnlessDBFeature('test_db_allows_multiple_connections')
```

Comment:
> because with sqlite3 the in-memory test database cannot be closed.

I said in the test for all backends, you can check that calling the method doesn't raise an exception.